<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>180</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WiFi Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelIP">
     <property name="text">
      <string>IPv4:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineIP"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="linePort"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="buttonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditClear"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
